### ISSUE TITLE
post_interactive_deposit - 'NoneType' object is not a mapping

### ISSUE THREAD

**Issue by yuriescl** [stellar/django-polaris#293]
Getting this error when submitting a form with invalid values in SEP-24 deposit:

```
Environment:


Request Method: POST
Request URL: http://localhost:8000/sep24/transactions/deposit/webapp/submit?transaction_id=cf800c29-483c-47ac-a15a-e4912efcef12&asset_code=PURPLE

Django Version: 2.2.15
Python Version: 3.8.2
Installed Applications:
['django.contrib.admin',
 'django.contrib.auth',
 'django.contrib.contenttypes',
 'django.contrib.sessions',
 'django.contrib.messages',
 'django.contrib.staticfiles',
 'phonenumber_field',
 'polaris',
 'app']
Installed Middleware:
['django.middleware.security.SecurityMiddleware',
 'polaris.middleware.PolarisSameSiteMiddleware',
 'django.contrib.sessions.middleware.SessionMiddleware',
 'django.middleware.locale.LocaleMiddleware',
 'corsheaders.middleware.CorsMiddleware',
 'django.middleware.common.CommonMiddleware',
 'django.middleware.csrf.CsrfViewMiddleware',
 'django.contrib.auth.middleware.AuthenticationMiddleware',
 'django.contrib.messages.middleware.MessageMiddleware',
 'django.middleware.clickjacking.XFrameOptionsMiddleware']



Traceback:

File "/home/yuri/.virtualenvs/venv/lib/python3.8/site-packages/django/core/handlers/exception.py" in inner
  34.             response = get_response(request)

File "/home/yuri/.virtualenvs/venv/lib/python3.8/site-packages/django/core/handlers/base.py" in _get_response
  115.                 response = self.process_exception_by_middleware(e, request)

File "/home/yuri/.virtualenvs/venv/lib/python3.8/site-packages/django/core/handlers/base.py" in _get_response
  113.                 response = wrapped_callback(request, *callback_args, **callback_kwargs)

File "/home/yuri/.virtualenvs/venv/lib/python3.8/site-packages/django/views/decorators/clickjacking.py" in wrapped_view
  50.         resp = view_func(*args, **kwargs)

File "/home/yuri/.virtualenvs/venv/lib/python3.8/site-packages/django/views/decorators/csrf.py" in wrapped_view
  54.         return view_func(*args, **kwargs)

File "/home/yuri/.virtualenvs/venv/lib/python3.8/site-packages/django/views/generic/base.py" in view
  71.             return self.dispatch(request, *args, **kwargs)

File "/home/yuri/.virtualenvs/venv/lib/python3.8/site-packages/rest_framework/views.py" in dispatch
  505.             response = self.handle_exception(exc)

File "/home/yuri/.virtualenvs/venv/lib/python3.8/site-packages/rest_framework/views.py" in handle_exception
  465.             self.raise_uncaught_exception(exc)

File "/home/yuri/.virtualenvs/venv/lib/python3.8/site-packages/rest_framework/views.py" in raise_uncaught_exception
  476.         raise exc

File "/home/yuri/.virtualenvs/venv/lib/python3.8/site-packages/rest_framework/views.py" in dispatch
  502.             response = handler(request, *args, **kwargs)

File "/home/yuri/.virtualenvs/venv/lib/python3.8/site-packages/rest_framework/decorators.py" in handler
  50.             return func(*args, **kwargs)

File "/home/yuri/.virtualenvs/venv/lib/python3.8/site-packages/polaris/sep24/utils.py" in wrapper
  40.                 return view(request)

File "/home/yuri/.virtualenvs/venv/lib/python3.8/site-packages/polaris/sep24/deposit.py" in post_interactive_deposit
  148.         scripts = registered_scripts_func({"form": form, **content})

Exception Type: TypeError at /sep24/transactions/deposit/webapp/submit
Exception Value: 'NoneType' object is not a mapping
```

The error is caused by `content` being `None` when `**content` gets evaluated.

**Comment by JakeUrban** [comment]
Thanks, will be included in the 1.0 release.

**Comment by JakeUrban** [comment]
https://github.com/stellar/django-polaris/commit/5054bf56b106684fc95d3ba8d2d879c54a79bbaa
